### DISCUSSION TITLE
A section to test the settings configured in the app.

### DISCUSSION THREAD

**Discussion by Ru5h4d** [florisboard/florisboard#1412] [Feedback]
Hi. Just started using FlorisBoard and I like the amount of customisation it provides. Can we add a section or an option to try out the settings? Thanks. :-)

**Comment by patrickgold** [comment]
Based on your comment I think you are on 0.3.13. If you want you can try out the latest beta06 release, which has a completely new preference screen (including a text box to try out the keyboard within the prefs!)

**Reply by Ru5h4d** [reply]
Yes, you are right. I didn't know it was already in the works. I'll try it out. Thanks. 👍🏼
